test/pseudo-tty/test-stderr-stdout-handle-sigwinch.js

```diff
@@ -28,3 +28,11 @@ process.stdout._refreshSize = wrap(originalRefreshSizeStdout,
                                    'calling stdout._refreshSize');
 
 process.emit('SIGWINCH');
+
+// In AIX, the child exits even before the python parent
+// can setup the readloop. Provide a reasonable delay.
+if (common.isAix) {
```

Comment:
for these, would it be better to delay the entire test when running on AIX?

e.g. something like...

```js
function doTest() {
  /** do the test stuff here **/
}
if (common.isAix)
  setTimeout(doTest, 200);
else
  doTest();
```